admin/package.json

```diff
@@ -7,6 +7,7 @@
     "npm": ">=8.3.1"
   },
   "dependencies": {
+    "@messageformat/core": "^3.2.0",
```

Comment:
I think we have an opportunity for some more refactoring here!

I was surprised that you updated messageformat but didn't have to change an import somewhere in admin js. Looking through the code I couldn't find it used anywhere! If I'm right I think you can...

1. Uninstall @messageformat/core from admin
2. That led me down the rabbit hole, because the only mention of "messageformat" in admin is in the `formatDate` service, and only for the `getTaskDueDate` which I don't think is used in admin at all and is a hangover from when we shared the code between webapp and admin. So this function can be deleted. It might be worth checking if any other functions in that service are likewise unrequired, and cleaning them up too!
3. That means we no longer need to load the `angular-translate-interpolation-messageformat` module in the main.js of admin so delete that.
4. _That_ means we don't need to do the weird inclusion step in the `browserify-admin` script so delete that line too.

I might be wrong of course, and you might find that it _is_ actually being used in admin but I can't see how if we never use the `messageformat` interpolation id. Needs testing to confirm, but if you do manage to do all 4 steps it'll really help clean up admin!